cli/cmd/apply_cmd.go

```diff
@@ -0,0 +1,60 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/kubeshop/tracetest/cli/actions"
+	"github.com/kubeshop/tracetest/cli/analytics"
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+var definitionFile string
+
+var applyCmd = &cobra.Command{
+	Use:    "apply [resource type]",
+	Short:  "Apply (create/update) resources to your Tracetest server",
+	Long:   "Apply resources",
```

Comment:
The long version is shorter than the short version, hehehehe

Maybe we can set both as the same text for now:

```suggestion
	Short:  "Create or update a resource to your Tracetest server",
	Long:   "Create or update a resource to your Tracetest server",
```